### PR TITLE
Add news item about direct link to assessments

### PR DESCRIPTION
**Pull request by jonatanschroeder**

### CODE REVIEW

File: news_items/026_direct_link/index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+     padding-bottom: 0em;
+ }
+</style>
+
+<p>
+  <a href="https://github.com/jonatanschroeder">Jonatan Schroeder</a> [INSERT NOTE ABOUT AUTHOR]. It is now easier to link to individual course instances and assessments on PrairieLearn. Instructors can now open the "Settings" menu for a course instance or assessment and find a URL that can be shared with students:
+</p>
+
+<p>
+  <img src="assessmentSettings.png">
```
Comment (mwest1066) [inline news_items/026_direct_link/index.html:27]: This image shows the Student Link to an `assessment_instance`, but I think the actual link should be to an `assessment`?

Comment (jonatanschroeder) [reply news_items/026_direct_link/index.html:27]: Fixed

**Review by mwest1066 — APPROVED** [review]

**Comment by mwest1066** [comment]
@jonatanschroeder do my changes look ok to you?

**Comment by jonatanschroeder** [comment]
> @jonatanschroeder do my changes look ok to you?

Yup, all good.

**Pull request merged**